SPECS/slirp4netns/slirp4netns.spec

```diff
@@ -0,0 +1,254 @@
+%global git0 https://github.com/rootless-containers/%{name}
+%global commit0 4367de7c3361c344155220a4e999ffd7432dad81
+%global shortcommit0 %(c=%{commit0}; echo ${c:0:7})
+# Used for comparing with latest upstream tag
+# to decide whether to autobuild (non-rawhide only)
+%define built_tag v1.1.8
+%define built_tag_strip %(b=%{built_tag}; echo ${b:1})
+%define download_url %{git0}/archive/%{built_tag}.tar.gz
+Summary:        slirp for network namespaces
+Name:           slirp4netns
+Version:        1.1.8
+Release:        3%{?dist}
+License:        GPLv2 AND MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            %{git0}
+Source0:        %{download_url}#/%{name}-%{version}.tar.gz
+BuildRequires:  autoconf
+BuildRequires:  automake
+BuildRequires:  gcc
+BuildRequires:  git
+BuildRequires:  glib2-devel
+BuildRequires:  go-md2man
+BuildRequires:  libcap-devel
+BuildRequires:  libseccomp-devel
+BuildRequires:  libslirp-devel
+BuildRequires:  make
+# no go-md2man in ppc64
+ExcludeArch:    ppc64
+
+%description
+slirp for network namespaces, without copying buffers across the namespaces.
+
+%package devel
+Summary:        %{summary}
+BuildArch:      noarch
+
+%description devel
+%{summary}
+
+This package contains library source intended for
+building other packages which use import path with
+%{import_path} prefix.
+
+%prep
+%autosetup -Sgit -n %{name}-%{built_tag_strip}
+
+%build
+./autogen.sh
+./configure --prefix=%{_usr} --libdir=%{_libdir}
+make generate-man
+
+%install
+make DESTDIR=%{buildroot} install install-man
+
+%check
+
```

Comment:
We might as well remove it, since it's empty.
```suggestion
```

Reply:
Have retained this.